.env.example

```
@@ -41,3 +41,7 @@ SENTRY_PROJECT = <SENTRY_PROJECT>
 SENTRY_ORG = <SENTRY_ORG>
 NEXT_PUBLIC_SENTRY_DSN = <SENTRY_DSN>
 SENTRY_AUTH_TOKEN = <SENTRY_AUTH_TOKEN>
+
+# Podcast Index API
+PODCAST_INDEX_AUTH_KEY = <PI_AUTH_KEY>
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)